### ISSUE TITLE
Get controller input

### ISSUE THREAD

**Issue by kabrio** [stalgiag/p5.xr#106]
I was wondering if it is possible to get input from the VR systems controllers?
If not, would it be possible to use the library together with the [A-Frame lib](https://aframe.io/) which does allow to register user input?

**Comment by kabrio** [comment]
Hmm, was hoping that p5js `mousePressed` or `touchStarted` might work here, but it seems not.
Any other ideas how to get user input from VR controllers?

**Comment by stalgiag** [comment]
Hi @kabrio this is a feature that would be fairly easy to implement but I have been letting this project sleep for a little while as I waited for iOS to get patched. I don't have an oculus quest so this isn't super easy for me to add this at the moment. Would love some help with adding them if you are interested. As a quick workaround in your own code, you can access the `session` after the canvas is made with `p5xr.instance.xrSession` then you can attach the event listener like below :

```
p5xr.instance.xrSession.addEventListener("select", onSelect);
```

**Comment by kabrio** [comment]
Thank you @stalgiag, I appreciate your input, helps a lot! I will see if I get this to work!

**Comment by kabrio** [comment]
This works great!
I managed to get input from my Quest 1 by creating the event listeners in setup() 

```
  let session = p5xr.instance.xrSession;
  session.addEventListener("select", onTrigger);
  session.addEventListener("squeeze", onGrip);
```
  

and then defining the functions (onTrigger and onGrip) later in the code.

**Comment by stalgiag** [comment]
closed by #115
